Update twelve-words.mdx

waveform06 opened the pull request:
updated with wallet use depreciated text

waveform06 committed:
Update twelve-words.mdx

updated with wallet use depreciated text

jthiller commented on docs/wallets/twelve-words.mdx:
```diff
@@ -10,20 +10,19 @@ slug: /wallets/twelve-words
 # Twelve Word Seed Phrase
 
-When setting up a Helium wallet through the Helium Hotspot app, you are prompted to write down the
+When creating a Helium wallet through the Helium Hotspot app, you were prompted to write down the
```
why are we mixing tenses
creating = active
were = past

samgutentag committed:
consolidating 12/24-word docs to “seed phrase” docs

link fixes
netlify redirects
page cleanup
“current phase” phrasing
sidebar updates
